astropy/table/tests/test_operations.py

```diff
@@ -1260,6 +1260,33 @@ def test_vstack_different_representation(self):
         with pytest.raises(ValueError, match='representations are inconsistent'):
             table.vstack([t1, t3])
 
+    def test_vstack_structured_column(self):
+        """Regression tests for gh-13271."""
+        # Two tables with matching names, including a structured column.
+        t1 = Table([np.array([(1., 1), (2., 2)], dtype=[('f', 'f8'), ('i', 'i8')]),
+                    ['one', 'two']], names=['structured', 'string'])
+        t2 = Table([np.array([(3., 3), (4., 4)], dtype=[('f', 'f8'), ('i', 'i8')]),
```

Comment:
What happens for a table with compatible but not identical structured columns? E.g. where the second one is `dtype=[('f', 'f4'), ('i', 'i4')])`.